test-suite/tests/compact-p002-out.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@context": {
+    "ex": {"@id": "http://example.org/"}
+  },
+  "@id": "http://example.org/id1",
+  "@type": ["http://example.org/Type1", "http://example.org/Type2"],
+  "http://example.org/term": {"@id": "http://example.org/id2"}
+}
```

Comment:
@gkellogg I see. I wonder if using "simple" as the arbiter here (as in key + string value) is too brittle. I'll move my thinking into the main thread...